src/test/java/core/basesyntax/services/FileReaderServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.services.impl.FileReaderServiceImpl;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceTest {
+    private static FileReaderService fileReaderService;
+    private static final String VALID_FILE_PATH = "src/test/java/resources/test.csv";
+    private static final String INVALID_FILE_PATH = "src/test/java/resources/InvalidTest.csv";
```

Comment:
Can we consider this as a wrong path, because there is a non-existent file, not a directory.
Perhaps this is a topic for conversation)